packages/OmegaPrint-Tests.package/TestComment.class/README.md

```diff
@@ -0,0 +1,7 @@
+TestComment is a class which holds some tests concerning comments. It tests from different starting points to see whether comments are removed in between or after statements but also before, after and in between a method.
```

Comment:
nach dem in fehlt ein Komma oder?

Reply:
Und ich würde wenn wir vorher in, between und after sschreiben, dann auch before, between und after schreiben von der Reihenfolge her.

Reply:
+1